examples/pligos/configs/gogetvanity/values.yaml

```diff
@@ -0,0 +1,70 @@
+probe:
+  - name: nginx
+    definition:
+      httpGet:
+        path: /healthz
+        port: 80
+      initialDelaySeconds: 20
+      periodSeconds: 10
+
+route:
+  - name: http
+    containerPort: 80
+    port: 80
+    protocol: TCP
+    path: /dgx
+
+image:
+  - name: nginx
+    registry: docker.io
+    repository: nginx
+    tag: 1.15.9-alpine
+    pullPolicy: Always
+
+command: []
+
+mount:
+  - name: nginxconfig
+    containerPath: /etc/nginx
+    definition:
+      configMap:
+        name: '{{ template "fullname" . }}'
+        items:
+          - key: nginx.conf
+            path: nginx.conf
+  - name: vanityconfig
+    containerPath: /www/golang
+    definition:
+      configMap:
+        name: '{{ template "fullname" . }}'
+        items:
+          - key: dgx.html
+            path: dgx.html
+
+resource:
+  - name: default
+    definition:
+      requests:
+        cpu: 5m
+
+configuration:
+  - name: gogetvanity
+    values:
+      hostname: realcloud.tech
+
+container:
+  - name: nginx
+    image: nginx
+    resource: default
```

Comment:
`resource:` -> `resources:`?
Not sure if this follows the standard that you used for routes or mounts. E.g.: you only have one route [http], but you assume that this can be a repeated value. Is this not the same case for resources? If it isn't sorry :)

Reply:
well, `resource` here is short for `resource definition` but I can see how this is confusing. I am not using the plural since it's referencing a single `resource definition` instance. I am thinking of a better name.